third_party/terraform/resources/resource_cloudfunctions_function.go

```diff
@@ -28,6 +28,14 @@ var functionAllowedMemory = map[int]bool{
 
 const functionDefaultAllowedMemoryMb = 256
 
+var functionAllowedIngressSettings = map[string]bool{
+	"INGRESS_SETTINGS_UNSPECIFIED": true,
```

Comment:
Would anyone realistically set the field to `INGRESS_SETTINGS_UNSPECIFIED`? If not, may as well leave it out.

Reply:
sure i leave it out